tests/lora/test_lora_layers_old_backend.py

```diff
@@ -1087,7 +1129,7 @@ def test_lora_fuse_nan(self):
         # corrupt one LoRA weight with `inf` values
         with torch.no_grad():
             sd_pipe.unet.mid_block.attentions[0].transformer_blocks[0].attn1.to_q.lora_layer.down.weight += float(
-                "inf"
+                "NaN"
```

Comment:
The reason why we do this is the following:

```python
import torch 

a = torch.tensor([float("inf")])
print(torch.isnan(a).any().item())
```

Prints False, whereas replacing the "inf" with "NaN" prints True.

Reply:
Ok for me, but even adding an "inf" value should yield NaNs for the forward pass